code/game/objects/structures/curtains.dm

```diff
@@ -1,3 +1,71 @@
+//
+// Curtain types declaration
+//
+/decl/curtain_kind
+	var/name = "curtain"
+	var/color = "white"
+	var/alpha = 255
```

Comment:
I feel like it might be more appropriate to grab alpha from the material decl (think it's `material.opacity` or something like that) since that would remove the need for a handful of these subtypes.

Reply:
Well, the thing is you got some plastic curtains that are transparent and some that aren't. Like the privacy curtain and medical ones. So its not really dependent on material. Its like you  can have cloth curtains with gaps in the cloth that makes them see-through. Its more of a factor of the way the material is used than its innate properties. At least I think.